WalletLibrary/WalletLibrary/Extensions/Token/JwsToken+Validation.swift

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * Extension for handling JSON Web Token operations.
+ */
+extension JwsToken
+{
+    /// Represents the DID (Decentralized Identifier) and Key ID extracted from the token header's key identifier.
+    struct TokenHeaderKeyId
+    {
+        let did: String
+        
+        let keyId: String
+    }
+    
+    /// Extracts and separates the DID and Key ID from the token header's key identifier.
+    /// - Returns: A `TokenHeaderKeyId` containing the DID and Key ID if the extraction is successful; otherwise, `nil`.
+    func getKeyId() -> TokenHeaderKeyId?
+    {
+        guard let components = headers.keyId?.split(separator: "#",
+                                                    maxSplits: 1,
+                                                    omittingEmptySubsequences: true),
+              components.count == 2 else
+        {
+            return nil
+        }
+        
+        return TokenHeaderKeyId(did: String(components[0]),
+                                keyId: String(components[1]))
+    }
+    
+    /// Validates the `iat` (Issued At) and `exp` (Expiration Time) claims of the token.
+    func validateIatAndExp() throws
```

Comment:
Can this method be configured to check for supplied validity period instead of hardcoding to 5 minutes?

Reply:
okay, I might change the name of this method as well.